contrib/download-frozen-image-v2.sh

```diff
@@ -79,9 +79,9 @@ fetch_blob() {
 			return 1
 		fi
 
-		curl -fSL "${curlArgs[@]}" \
-			"$blobRedirect" \
-			-o "$targetFile"
+		echo   "$blobRedirect"  output  "$targetFile"
+		#use wget to download file with continuously transmission
+		wget -c -O "$targetFile"  "$blobRedirect"
```

Comment:
This looks to be the equivalent of `-C -` with cURL; https://curl.se/docs/manpage.html#-C

```markdown
-C, --continue-at <offset>

  Continue/Resume a previous file transfer at the given offset. The given offset
  is the exact number of bytes that will be skipped, counting from the beginning
  of the source file before it is transferred to the destination.

  (...)

  Use "-C -" to tell curl to automatically find out where/how to resume the transfer.
  It then uses the given output/input files to figure that out.

  (...)

Examples:

 curl -C - https://example.com
 curl -C 400 https://example.com
```